Dockerfile

```diff
@@ -5,12 +5,21 @@ MAINTAINER  Mark Musante mark.musante@gmail.com
 WORKDIR /ifcomp-build
 ADD ./IFComp/cpanfile /ifcomp-build
 
+RUN apt-get update && apt-get install -y apt-utils
```

Comment:
The Docker guides I've read say that as much as possible multiple RUN commands should be combined together.

Reply:
i know from experience that splitting them makes it faster to generate new docker images, so when I was developing the file it made it easier to incrementally build up the changes I needed

But I will take a look for some best-practices guides and see what we can do about cleaning it up

Reply:
There could be different guidelines based on whether you're developing an image for others to use, or whether it's just for yourself?